.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,9 +1,27 @@
-Please delete this line and the text below before submitting your contribution.
+# Pull Request checklist
 
----
+Thanks for contributing!
+Please keep in mind that lesson maintainers are volunteers and it may be some time before they can respond to your contribution. Although not all contributions can be incorporated into the lesson materials, we appreciate your time and effort to improve the curriculum.
 
-Thanks for contributing! If this contribution is for instructor training, please send an email to checkout@carpentries.org with a link to this contribution so we can record your progress. You’ve completed your contribution step for instructor checkout just by submitting this contribution.  
+* * *
 
-Please keep in mind that lesson maintainers are volunteers and it may be some time before they can respond to your contribution. Although not all contributions can be incorporated into the lesson materials, we appreciate your time and effort to improve the curriculum.  If you have any questions about the lesson maintenance process or would like to volunteer your time as a contribution reviewer, please contact The Carpentries Team at team@carpentries.org.  
+## Tasks
 
----
+- [ ] Read [CONTRIBUTING.md](https://github.com/ESMValGroup/tutorial/blob/master/CONTRIBUTING.md).
+- [ ] Create an [issue](https://github.com/ESMValGroup/tutorial/issues) to discuss what you are going to do.
+- [ ] Give this pull request a descriptive title.
+- [ ] If you are contributing to the lesson materials, please make sure the content conforms to the `Lesson development` section in [CONTRIBUTING.md](https://github.com/ESMValGroup/tutorial/blob/master/CONTRIBUTING.md) and does not contain any spelling or grammatical errors.
+- [ ] Preferably Codacy code quality checks pass. However, a few remaining hard to solve Codacy issues are still acceptable. Status can be seen below your pull request. If there is an error, click the link to find out why. If you suspect Codacy may be wrong, please ask by commenting.
```

Comment:
This seems a bit confusing, as this repository does not contain code? Or does it?

Reply:
@bouweandela My apologies. I corrected this item. The codacy checks contain a markdown linter.